docs/proxy/residential_proxy.md

```diff
@@ -19,7 +19,31 @@ Residential proxies support [IP address rotation]({{@link proxy.md#ip-address-ro
 
 **Pricing is based on data traffic**. It is measured for each connection made and displayed on your [dashboard](https://console.apify.com) in the Apify console.
 
-If you would like to use residential proxy or for more information, [contact us](https://apify.com/contact).
+## [](#limitations) Limitations
+
+Apify provides 2 levels of residential proxy:
+
+| Level | Availability | Supported domains | Other limitations |
+|-------|--------------|-------------------|-------------------|
+| Restricted residential proxy | Every user | >500 domains | Requires man in the middle access to connection |
+| Full residential proxy | Enterprise level accounts | Entire web| None |
+
+### Restricted residential proxy
+
+Restricted residential proxy is available for all the users with some limitations.
+
+First of all, restricted residential proxy supports only certain domains and paths. The pool of 500 domains is increased every month and covers the most frequent use cases.
```

Comment:
> The pool of 500 domains is increased every month and covers the most frequent use cases.

Does this mean the user starts out with 500 domains and then gets more?

Reply:
Nope that the new domains are being enabled in the system globally

Reply:
So then you'll still be able to crawl 500 different domains but the overall list will be bigger?